packages/edit-site/src/components/editor/index.js

```diff
@@ -38,6 +38,7 @@ import Header from '../header';
 import { SidebarComplementaryAreaFills } from '../sidebar';
 import BlockEditor from '../block-editor';
 import KeyboardShortcuts from '../keyboard-shortcuts';
+import GlobalStylesProvider from './global-styles-provider';
```

Comment:
I highly recommend reviewing this file by activating the "Hide whitespace changes", otherwise it's very difficult to understand the changes here. For reference, this adds two new providers within the tree and doesn't modify anythingelse.